web/src/pages/Settings/Presets/CreatePresetModal/CreatePresetModal.jsx

```diff
@@ -123,17 +125,34 @@ const StructuredDictAsTree = ({ dict, depth = 0, path = [] }) => {
     };
 
     let entries = [];
-    let i = 0;
-    for (const d of dict.content) {
-        entries.push(
-            <StructuredDictAsTree
-                key={i}
-                dict={d}
-                depth={depth + 1}
-                path={[...path, i]}
-            />
-        );
-        i += 1;
+    if(is_streamer){
+        let i = 0;
+        for (const d of dict.content) {
+            if (depth == 0){  // depth=0 only applies to the 4 streamer outputs. This is necessary so that the tree can still save the state of the Stream checkbox without displaying it
+                entries.push(
+                    <StructuredDictAsTree
+                        key={i}
+                        dict={d}
+                        depth={depth + 1}
+                        path={[...path, i]}
+                    />
+                );
+            }
+            i += 1;
+        }
+    } else {
+        let i = 0;
+        for (const d of dict.content) {
+            entries.push(
+                <StructuredDictAsTree
+                    key={i}
+                    dict={d}
+                    depth={depth + 1}
+                    path={[...path, i]}
+                />
+            );
+            i += 1;
+        }
```

Comment:
```suggestion
    let entries = []; 
    let i = 0;
    for (const d of dict.content) {
        let e = <StructuredDictAsTree
            key={i}
            dict={d}
            depth={depth + 1}
            path={[...path, i]} 
        >   
        if(!is_streamer || (is_streamer && depth === 0)) {
            entries.push(e)
        }   
        i += 1;
    }
```

try something like this instead (it's untested)

Reply:
It is now tested, works like a charm :+1: